Mail/Views/Menu Drawer/Folders/FolderCell.swift

```diff
@@ -154,13 +154,7 @@ struct FolderCellContent: View {
     @ViewBuilder
     private var background: some View {
         if cellType == .link {
-            SelectionBackground(
-                isSelected: isCurrentFolder,
-                offsetX: 8,
-                leadingPadding: 0,
-                verticalPadding: 0,
-                defaultColor: MailResourcesAsset.backgroundMenuDrawer.swiftUiColor
-            )
+            SelectionBackground(isSelected: isCurrentFolder, offsetX: 8, leadingPadding: 0, verticalPadding: 0)
```

Comment:
We could use default values instead of passing the same arguments everywhere.

Reply:
Great idea